packages/esm-patient-medications-app/src/config-schema.ts

```diff
@@ -45,4 +50,5 @@ export interface ConfigObject {
     display: string;
   };
   drugOrderTypeUUID: string;
+  mapOrderEncounterToCurrentVisit: boolean;
```

Comment:
Is this typing meant for `config.currentVisitRequired`? If so we need to align on the naming.